Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1744,7 +1741,13 @@ void TextureCacheBase::CopyRenderTargetToTexture(u32 dstAddr, EFBCopyFormat dstF
       entry->may_have_overlapping_textures = false;
       entry->is_custom_tex = false;
 
-      CopyEFBToCacheEntry(entry, is_depth_copy, srcRect, scaleByHalf, dstFormat, isIntensity);
+      // For XFB, the resulting XFB copy texture is the height of the actual texture + the y-scaling
+      // however, the actual Wii/GC texture that we want to "stretch" to this copy is without the y-scaling
+      // so we need to remove it here
+      // The best use-case for this is PAL games which have a different aspect ratio
+      EFBRectangle src_rect = srcRect;
```

Comment:
This should *probably* be given a name different from the one `srcRect` should actually be ;)